testui/elements/testui_element.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import annotations
```

Comment:
Why is this needed?

Reply:
to annotate methods that return self.

Reply:
I think you can write them like this and not use this `__future__` import:
```py
def clear(self) -> "Elements":
```